e2e/README.md

```diff
@@ -0,0 +1,41 @@
+
+## Usage
+
+Install dependencies
+```
+python3
+kustomize   # MUST BE VERSION 3.2.0
```

Comment:
this has to be precise this or >=3.2.0 ?

Reply:
Must be precise due to changes in kustomize from newer versions that affects resource creation order. This needs to be updated in the other READMEs as well, will create an issue.

Reply:
If it has to be updated in README for customer please make it as p0 @surajkota @mbaijal fyi.